docs/components/course/course-edit/course-content.md

```diff
@@ -0,0 +1,51 @@
+CourseContent 
+===============
+This component wraps components needed for content editing in a course. It provides a container or layout for these components. 
+
+## Structure
+### Dependencies
+* `CourseFiles`from `@/components/course/course-edit/course-files.vue` 
+* `stripKey`: from `@/mixins/general/helpers`
+* `vuex`
+
+### Components
+- `CourseFiles`
+
+### Mixins
+* `locale`
+* `routes`
+
+Component Functionality
+---------
+
+### Props
+- `type`: a string type with a default value of an empty string
+- `saveChanges`: a boolean type with a default value of false.
+
+### Data
+- `preview`: is initialized with a boolean value of false.
+
+### Computed Properties
+#### VueX Getters
+
+- `courseContent`: 
+
+#### Local
+- `cid`: returns the type of content
+- `comps`: return create, edit, view components for given content type
+- `editContent`: returns true if it's editing an existing content 
+- `stepData`: return input data for content 
+
+### Watchers
+- `saveChange`:is set to true, it calls the "save" method to save the edited content.
```

Comment:
a bit misleading: Would rephrase to "when set to true...", also missing backticks